provider/win_driver_utils.py

```diff
@@ -180,3 +192,31 @@ def copy_file_to_samepath(session, test, params):
         if status != 0:
             test.error("Copy file error,"
                        " the detailed info:\n%s." % output)
+
+
+def memory_leak_check(vm, test, params):
+    """
+    Uninstall the driver and check if a memory leak will occur
```

Comment:
We could have more explanation here, otherwise people will be confused why it is called `memory_leak_check` but actually only uninstall and reinstall drivers.

Reply:
Okay!